web/blog/vast-v2.3/index.md

```diff
@@ -96,7 +96,7 @@ vast:
         partition-index: false
         # Don't create partition indexes for fields of type addr.
```

Comment:
```suggestion
        # Don't create partition indexes for IP address fields.
```

Reply:
I don't think we should update older blog posts.